vortex-mask/src/lib.rs

```diff
@@ -98,6 +98,17 @@ pub enum Mask {
     Values(Arc<MaskValues>),
 }
 
+impl Mask {
+    /// Convert the mask into a vector of boolean values.
+    pub fn to_vec(&self) -> Vec<bool> {
```

Comment:
Maybe we should `#[cfg(test)]` since this is crazy slow

Reply:
I have been using a test-harness feature for functions like this